pkg/admission/clusterworkspacetypeexists/admission_test.go

```diff
@@ -1551,9 +827,70 @@ func (b builder) allowingParent(qualifiedName string) builder {
 	return b
 }
 
+func (b builder) allowingChild(qualifiedName string) builder {
```

Comment:
no pointer receiver/return? is this by purpose? effectively, we're mutating the builder.

Reply:
the only field in builder is a pointer. So this is essentially `type builder *T`. With the later we would use a receiver pointer. But here, double pointers don't make sense.